README.md

```diff
@@ -55,7 +56,7 @@ packages.
 
  * Save and exit.
 
- * Run `make`
+ * Run `make` (for a real benchmarking setup do *not* use `-j`)
```

Comment:
Because it makes the build less repeatable. It's a long shot I know, but we also didn't use `-j` in `build.sh` in the warmup experiment.